english/melt-wall/melt-wall.md

```diff
@@ -0,0 +1,2 @@
+You pick up the lighter and try to use it to melt the
```

Comment:
If you put these new files inside the `lighter-use` directory that should fix your links, and I also think it's a nicer directory structure.